src/components/Common/FolderContextMenu.tsx

```diff
@@ -73,21 +73,21 @@ export const FolderContextMenu = ({
         display: !!onUpload && !isExternal,
         dataQa: 'upload',
         Icon: IconUpload,
-        onClick: onUpload,
+        onClick: onUpload as onClickMenuItemHandler,
```

Comment:
Is this assertion really needed? Maybe we need to have type which will not need type assertions? Create TODO: here in such case and create techdebt ticket to fix it in future